### ISSUE TITLE
:x closes Atom

### ISSUE THREAD

**Issue by JasonTheAdams** [lloeki/ex-mode#167]
Hey! Thanks for this. Makes atom feel more familiar. :)

One quirk though: When I use `:x` it closes atom entirely, not just the current file. The `:q` command works just fine, however.

**Comment by zephraph** [comment]
Yep. This was pretty annoying to me too. I kept accidentally doing that over and over. 

Here's how I fixed it. 
```coffeescript
# init.coffee
atom.packages.onDidActivatePackage (pack) ->
  if pack.name == 'ex-mode'
    Ex = pack.mainModule.provideEx()
    Ex.registerAlias 'x', 'wq'
```

**Comment by JasonTheAdams** [comment]
Nice! Thank you! Feels like it should be built into ex-mode, so I'll leave this open, but that's a great workaround.

**Comment by jtr109** [comment]
I got the problem too. Thanks for your solution. @zephraph

**Comment by jazzpi** [comment]
This should be fixed in v0.14.0, please update ex-mode and restart Atom.

**Comment by jtr109** [comment]
Great! It fixed in v0.14.0 by updating. Thanks a lot! @jazzpi
